charon/src/extract_global_assignments.rs

```diff
@@ -0,0 +1,74 @@
+//! Extracts globals from operands to put them in a separate let binding.
+//! This is done to increase sequentiality of LLBC and reduces the cases to handle for operands.
+//! It will ease the functional translation.
+//!
+//! Also extracts statics fom operands for the same reason and because we want to
+//! treat them as globals in LLBC.
+//! To do that, we add a new variable to reference the static :
+//! they are accessed by reference in MIR, whereas globals are accessed by value.
```

Comment:
Statics are accessed through a reference, so because we translate them to globals that are accessed by value, I also add a statement like `let static_var = &my_static`, and move `static_var` in the operand (which expect a reference).

Reply:
Ok, thanks, this explanation is clear. I think you could put it in the comment in the code :)

Reply:
Yup, I added it ^^